pkg/tcpip/link/fdbased/endpoint.go

```diff
@@ -287,9 +287,11 @@ func createInboundDispatcher(e *endpoint, fd int, isSocket bool) (linkDispatcher
 			// enable PACKET_FANOUT mode is the underlying socket is
 			// of type AF_PACKET.
 			const fanoutType = 0x8000 // PACKET_FANOUT_HASH | PACKET_FANOUT_FLAG_DEFRAG
-			fanoutArg := fanoutID | fanoutType<<16
-			if err := syscall.SetsockoptInt(fd, syscall.SOL_PACKET, unix.PACKET_FANOUT, fanoutArg); err != nil {
-				return nil, fmt.Errorf("failed to enable PACKET_FANOUT option: %v", err)
+			// This const will overflow a 32 bit int and won't compile. Variable
+			// type conversion from 32 bit uint to 32 bit int will overflow correctly.
+			var fanoutArg uint = (fanoutID | fanoutType<<16)
```

Comment:
Looks like this line is failing the build

/home/runner/.cache/bazel/_bazel_runner/000ef9ee90a3e625fcf1f4edcfe0fd56/sandbox/linux-sandbox/484/execroot/__main__/pkg/tcpip/link/fdbased/endpoint.go:292:8: cannot use fanoutID | fanoutType << 16 (type int) as type uint in assignment